.circleci/config.yml

```diff
@@ -26,6 +26,14 @@ jobs:
       #    command: ./scripts/run_in_ci.sh Rscript -e "rmarkdown::render('analyses/mutect2-vs-strelka2/01-set-up.Rmd', clean = TRUE);
       #                                                rmarkdown::render('analyses/mutect2-vs-strelka2/02-analyze-concordance.Rmd', clean = TRUE)"
      
+      - run:
+          name: Collapse RSEM
+          command: ./scripts/run_in_ci.sh Rscript analyses/collapse-rnaseq/01-summarize_matrices.R -e data/pbta-gene-expression-rsem-fpkm.stranded.rds -o data/pbta-gene-expression-collapsed-rsem-fpkm.stranded.rds
```

Comment:
The `-e` option here is not defined in `01-summarize_matrices.R`, causing CI to fail. Should be `-i`?

Reply:
the CI probably has the old example run. I updated all the scripts + input parameters. Example runs are on top of the R scripts.

Reply:
Can you update this file to correct this?

Reply:
@jashapiro @jaclyn-taroni The circleci on is failing because one of my scripts uses the gencode v27 gtf file. Currently, I am just referring to it under scratch/ but haven’t pushed it because it is quite big. Should we include that in the data release? If not then how do I circumvent the error?

Reply:
I agree that having a defined annotation file in the data download might be a good idea; we can suggest it to @jharenza for the next release. Alternatively, we do currently have annotations installed through [txdb](https://bioconductor.org/packages/release/data/annotation/html/TxDb.Hsapiens.UCSC.hg38.knownGene.html) but I do not know if those would have the information you require. 

If not, for now you could add the file to your analysis folder; At 39MB, it really isn't too big for github.

Reply:
The gtf is actually 1.1 GB. Do you still think it is a good idea to put on github? If yes, I can put it under `scratch/` folder and it should pass the test.

Reply:
That is uncompressed. If you include the gzipped version at ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_27/gencode.v27.primary_assembly.annotation.gtf.gz version it will only be 40MB.  You can't put it in scratch, as that is not synced to the repository, and won't be present in the CI image.

I see you are reading the gtf with `rtracklayer::import`, and while I don't know if that can read compressed files, many packages certainly can. If not, you could write a wrapper shell script that decompresses the file to scratch before running your R scripts. (You might want to do this anyway, to have only a single RUN statement in CI.)

Reply:
@jashapiro Thanks for the idea - gzipped worked! 

For now, I have the input gtf and the output  tables for `01-summarize_matrices.R` under `analyses/collapse-rnaseq`. It looks like finally all checks have passed so let me know once the code is reviewed and if there are any major changes to be made.